sw/device/lib/testing/test_framework/BUILD

```diff
@@ -41,19 +41,71 @@ cc_library(
 )
 
 ld_library(
-    name = "linker_script",
-    script = "ottf.ld",
+    name = "ottf_ld_common",
+    fragments = ["ottf_common.ld"],
+    deps = ["//sw/device:info_sections"],
+)
+
+ld_library(
+    name = "ottf_ld_silicon_creator_slot_a",
+    script = "ottf_silicon_creator_a.ld",
+    deps = [
+        ":ottf_ld_common",
+        "//hw/top_earlgrey/sw/autogen:top_earlgrey_memory",
```

Comment:
Should  this dependency be moved to `ottf_ld_common`?

Reply:
That is not directly used in `ottf_ld_common` and violates the ["Include What You Use" directions](https://google.github.io/styleguide/cppguide.html#Include_What_You_Use) of the style guide.